back-end/routes/main-route.js

```diff
@@ -1,8 +1,10 @@
 const express = require('express');
 const mainRouter = express.Router();
+const {getAllOrders,addNewOrder,deleteOrderById} = require("../controllers/main-controller");
```

Comment:
You need to add spaces after the commas. make sure to use prettier (VS Code extention) to format your code, it will do it automatically